lti_consumer/migrations/0006_add_on_model_config_for_lti_1p1.py

```diff
@@ -14,6 +21,12 @@ class Migration(migrations.Migration):
 
     operations = [
         migrations.AddField(
+            model_name='lticonfiguration',
+            name='config_id',
+            field=models.UUIDField(default=uuid.uuid4, blank=True, null=True),
+        ),
+        migrations.RunPython(create_config_ids),
+        migrations.AlterField(
             model_name='lticonfiguration',
             name='config_id',
             field=models.UUIDField(default=uuid.uuid4, editable=False, unique=True),
```

Comment:
should we leave the `default` arg here?

Reply:
I think it needs to be there since Django will otherwise detect it as a difference between the old migration state and the current code and re-create a migraiton.

Reply:
At least according to this [1] :shrug: 

- [1] https://docs.djangoproject.com/en/dev/howto/writing-migrations/#migrations-that-add-unique-fields